app/controllers/concerns/response.rb

```diff
@@ -0,0 +1,7 @@
+module Response
+  extend ActiveSupport::Concern
+
+  def json_response(object, status = :ok)
+    render json: object, status:
```

Comment:
- [ ] **[OPTIONAL]** Good job returning a random greeting object. However, I don't think it is a good idea to return all the database details of the record in an API call. I think it would be better to trim your response by returning only the `message` value, like so:
  ```
  ... json: object.message, ...
  ```
  ![mih-message](https://user-images.githubusercontent.com/36542986/177209239-2a5bc019-22e0-47b7-b799-6237c6e768cf.jpg)

  What do you think?

Reply:
@TedLivist I do agree with that, I didn't think much of it, thanks for the catch!